apps/backend/src/resolvers/mutations/AddTechnicalReviewMutation.ts

```diff
@@ -36,6 +36,9 @@ export class AddTechnicalReviewInput implements Partial<TechnicalReview> {
 
   @Field(() => String, { nullable: true })
   public files: string | null;
+

```

Comment:
I can see that when you call `addTechnicalReview` then it calls an `update` method in the datasource. I have two questions
1. How do you make sure that the review already exists?
2. Can't you use `technical_review_id` here instead of `proposalPK` and `instrumentId` combination?

Reply:
We are not sending the technical_review_id as an input argument even if it is an update. Maybe this needs a bit of a refactor later and make the update completely separated from create.